Lab meeting blog | 26 September 2019 (#90)
WhitakerLab/whitakerlab.github.io: _posts/blog/2019-09-26-2019-09-26-lab-meeting.md

Scroll down past all the amazing updates to hear more about **Ang** and **Yini**'s work on [a registered report to investigate the effects of polygenic risk for schizophrenia on brain structure](#a-registered-report-to-investigate-the-effects-of-polygenic-risk-for-schizophrenia-on-brain-structure).

## Celebrations and cool things to share

**Kirstie** is delighted to welcome [Malvika Sharan](https://www.software.ac.uk/about/fellows/malvika-sharan) to the lab as the community manager for [_The Turing Way_](https://github.com/alan-turing-institute/the-turing-way).
She's currently working one day per week as she finishes up her responsibilities at EMBL in Heidelberg and will be full time at the Turing in London mid-January.

**Malvika** is experiencing some imposter syndrome (aren't we all?) and is figuring out how to juggle her current work and wanting to get started with _The Turing Way_, but is overall really excited to join the lab!
She asked the lab members for local tips for new folks at the Turing.
They suggested:

* If you want fruit arrive early - it gets delivered around 8:30am and is gone by 9am!

> * The tea cubpoard may look like someone's secret stash but its actually available for everyone!

Reviewer: ```suggestion
* The tea cupboard may look like someone's secret stash but its actually available for everyone!
```

In fact, the first [Reproducible Research Lunch](https://github.com/alan-turing-institute/ReproducibleResearchResources/issues/19) at the Turing (starting next week) is on Hacktoberfest!
Everyone is welcome and Kirstie and Louise are looking for suggestions for future discussions.

**Christina** is currently on holiday and recommends travelling to St. Petersburg!

**Sarah** ran her first ["Learn Azure"](https://github.com/alan-turing-institute/learn-azure) session for the [Living with Machines](https://www.turing.ac.uk/research/research-projects/living-machines) group (and is now officially exhausted 😅).

**Yo** also shared the link to apply for the [2020 Software Sustainability Fellowships](https://software.ac.uk/news/applications-fellowship-programme-2020-are-now-open)!
Fellows are also awarded £3,000 to spend over 15 months to improve how research software is done in their domains and/or area of work.
She, **Malvika** and **Patricia** are fellows already and, although the money is fun, they'd recommend the fellowship for the networking and collaborations workshop alone!

> **Louise** is running a [two-day workshop](https://www.turing.ac.uk/events/introduction-spark-data-scientists) at the Turing  introducing [Spark](https://spark.apache.org/) at the end of October.

Reviewer: ```suggestion
**Louise** is helping to organise a [two-day workshop](https://www.turing.ac.uk/events/introduction-spark-data-scientists) at the Turing  introducing [Spark](https://spark.apache.org/) at the end of October.
```

**Elizabeth** shared [ImageNet Roulette](https://imagenet-roulette.paglen.com).
(It was only available until Friday 27 September, so that link will take you to a write up of the project by authors [Kate Crawford](https://katecrawford.net) and [Trevor Paglen](http://www.paglen.com)).
She was pleasantly surprised that the public outcry led ImageNet to [scrub about half (!!) of their "people" category](https://www.msn.com/en-us/news/technology/everyday-people-how-a-massive-picture-database-sparked-a-discussion-about-ai-and-bias/ar-AAHy5Jy).

**Ang** has been exploring the [Pingouin](https://pingouin-stats.org) 🐧 Python package to perform standard statistical tests.
Checkout the [interactive demo on Binder](https://mybinder.org/v2/gh/raphaelvallat/pingouin/develop).

**Yini** recommends [NeuroPower](http://neuropowertools.org/neuropower/neuropowerstart) which helps researchers make sure they're appropriately powered (eg with a large enough sample size) before they start their experiments.

## Questions we're thinking about

> **Kirstie** has joined a large collaboration project to make sure that everything they do is reproducible from the start (the project will officially launch around February 2020) so she's thinking about ways to map members of the collaboration to ensure that everyone knows who is in the network and what information they have access to.

Reviewer: ```suggestion
**Kirstie** has joined a large collaborative project to make sure that everything they do is reproducible from the start (the project will officially launch around February 2020) so she's thinking about ways to map members of the collaboration to ensure that everyone knows who is in the network and what information they have access to.
```
